packages/hardhat-core/test/internal/artifacts.ts

```diff
@@ -3,12 +3,13 @@ import fsExtra from "fs-extra";
 import * as os from "os";
 import * as path from "path";
 
+import { beforeEach } from "mocha";
```

Comment:
?

Reply:
??

Reply:
I think this is an accidental auto-import. We don't do this normally (mainly because it prevents the file from being run stand-alone with mocha).